zppy/defaults/default.ini

```diff
@@ -106,28 +101,34 @@ input_component = string(default="")
 
 [ts]
 area_nm = string(default="area")
-cmip_metadata = string(default="inclusions/e3sm_to_cmip/default_metadata.json")
 # Days per file
 dpf = integer(default=30)
 extra_vars = string(default="")
 # Time-steps per day
 tpd = integer(default=1)
-ts_fmt = string(default="ts_only")
 # Model component having generated input files (eam, eamxx, elm, mosart, ...)
 input_component = string(default="")
 
   [[__many__]]
   area_nm = string(default=None)
-  cmip_metadata = string(default=None)
   dpf = integer(default=None)
   extra_vars = string(default=None)
   tpd = integer(default=None)
-  ts_fmt = string(default=None)
+  input_component = string(default=None)
+
+[e3sm_to_cmip]
+cmip_metadata = string(default="inclusions/e3sm_to_cmip/default_metadata.json")
+# Model component having generated input files (eam, eamxx, elm, mosart, ...)
+input_component = string(default="")
+
+  [[__many__]]
+  cmip_metadata = string(default=None)
   input_component = string(default=None)
 
 [tc_analysis]
 # NOTE: always overrides value in [default]
 input_files = string(default="eam.h2")
+# TODO for v3.0.0: Remove this parameter
```

Comment:
I made notes of things we should also change for the v3 release, but that don't fit in with the changes here. https://github.com/E3SM-Project/zppy/issues/653 will track these things.